apps/volontulo/tests/views/test_offers.py

```diff
@@ -181,6 +190,21 @@ def test_offers_create_get_method(self):
         self.assertEqual(response.status_code, 200)
         self.assertTemplateUsed(response, 'offers/offer_form.html')
 
+    def test_offers_create_no_organization_get_method(self):
+        u"""Test page for offer creation - tendering template with form."""
```

Comment:
There is a plan to not use `u` prefix here, because Python 3 is our main environment,